src/components/artefact/artefact-image.vue

```diff
@@ -2,13 +2,20 @@
     <artefact-svg v-if="loaded"
                   :artefact="artefact"
                   :aspect-ratio="aspectRatio">
-        <iiif-image 
-            v-for="imageSetting in visibleImageSettings"
-            :key="imageSetting.image.url"
-            :image="imageSetting.image"
-            :opacity="imageSetting.normalizedOpacity"
-            :boundingBox="boundingBox"
-            :maxWidth="maxWidth"/>
+        <g v-if="!this.artefact.isVirtual">
+            <iiif-image 
+                v-for="imageSetting in visibleImageSettings"
+                :key="imageSetting.image.url"
+                :image="imageSetting.image"
+                :opacity="imageSetting.normalizedOpacity"
+                :boundingBox="boundingBox"
+                :maxWidth="maxWidth"/>
+        </g>
+        <path v-if="artefact.isVirtual"
```

Comment:
You can also use `v-else`, but maybe the explicit check is somehow safer.

Reply:
I don't like `v-else` much, since it can break if I add another condition somewhere. Since ifs are not really nested, but rather just attributes, it can be hard to catch.

Reply:
That sounds fair to me.